source/com.google.android.gms/play-services-safetynet/Additions/Additions.cs

```diff
@@ -25,15 +25,24 @@ public static async Task<bool> ValidateWithGoogle (this SafetyNetApiAttestationR
             var content = new StringContent (jsonReq, Encoding.Default, "application/json");
 
             var response = await http.PostAsync (url + validationApiKey, content);
-            var data = await response.Content.ReadAsStringAsync ();
-
             response.EnsureSuccessStatusCode ();
 
-            var json = JsonObject.Parse (data);
-            if (json.ContainsKey ("isValidSignature"))
-                return json ["isValidSignature"].ToString ().Trim ('"').Equals ("true");
+            using var stream = await response.Content.ReadAsStreamAsync();
+            JsonDocument doc = await JsonDocument.ParseAsync(stream);
+            JsonElement root = doc.RootElement;
+            bool isValidSignature = false;
+
+            try
+            {
+                JsonElement signature = root.GetProperty("isValidSignature");
+            if (signature.GetBoolean())
+                isValidSignature = true;
```

Comment:
There is a `TryGetProperty()` you can use instead of try-catch:
```csharp
if (root.TryGetProperty ("isValidSignature", out var signature)) {
    return signature.GetBoolean();
} else {
    return false;
}
```